packages/wallet/test/services/HandlesTracker.test.ts

```diff
@@ -33,15 +45,19 @@ const expectedHandleInfo = {
 describe('createHandlesTracker', () => {
   it('matches utxo$ assets to given handlePolicyIds and adds context from assetInfo$ and tip$ observables', () => {
     createTestScheduler().run(({ hot, expectObservable }) => {
-      const utxo$ = hot(     '-ab-', { a: utxo, b: utxo2 });
-      const tip$ = hot(      'a-b-', { a: ledgerTip, b: ledgerTip2 });
+      const utxo$ = hot('-ab-', { a: utxo, b: utxo2 });
+      const tip$ = hot('a-b-', { a: ledgerTip, b: ledgerTip2 });
       const assetInfo$ = hot('-a-b', {
         a: new Map(),
         b: new Map([[handleAssetId, handleAssetInfo]])
       });
 
       const handles$ = createHandlesTracker({
-        assetInfo$, handlePolicyIds: [handlePolicyId], logger, tip$, utxo$
+        assetInfo$,
```

Comment:
This formatting change should also be governed with lint rules, so I don't expect this to change unless the lint rules are

Reply:
b666699d85e862981740c6a00a3321c1b1bd1154 should solve this